### PR TITLE
Implement bingbong voice command

### PR DESCRIPTION
**Pull request by a-gavin**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,2 +1,3 @@
 *.wav filter=lfs diff=lfs merge=lfs -text
 *.mp3 filter=lfs diff=lfs merge=lfs -text
+assets/sounds/general/bingBong.mp3 filter=lfs diff=lfs merge=lfs -text
```
Comment (greenerino) [inline .gitattributes]: Is this added separately from the rest of the mp3 files? Why does line 2 not include this?

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM node:latest
+FROM node:16.6.0
```
Comment (greenerino) [inline Dockerfile:1]: Should this be a separate PR?

File: .gitattributes (hunk 1 above)
Comment (a-gavin) [reply .gitattributes]: Good point. I'll change that ; )

File: .gitattributes [hunk 3]
```diff
@@ -1,2 +1 @@
-*.wav filter=lfs diff=lfs merge=lfs -text
-*.mp3 filter=lfs diff=lfs merge=lfs -text
+assets/sounds/** filter=lfs diff=lfs merge=lfs -text
```
Comment (greenerino) [inline .gitattributes:1]: image

**Review by greenerino — APPROVED** [review]
Looks good! Was in a call with Alex and he showed me that it worked. Please wait for the DUCCster to review before merging

**Pull request merged**